README.md

```diff
@@ -214,23 +214,17 @@ You'll see the `plugins.cookie` section, which is loading `examples/plugin/nemo-
 'use strict';
 
 module.exports = {
-  "setup": function (nemo, callback) {
+  "setup"(nemo, callback) {
```

Comment:
Okay. Disregard these comments.  I guess it was the "quotes" around the function name that was throwing me off.  I guess these changes are okay.

https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=env&targets=&browsers=&builtIns=false&debug=false&code='use strict'%3B%0A%0Amodule.exports %3D {%0A setupa(nemo%2C callback) {}%2C%0A "setupb"(nemo%2C callback) {}%2C%0A setupc%3A (nemo%2C callback) %3D> {}%0A}%3B